ec2api/db/sqlalchemy/api.py

```diff
@@ -200,6 +200,16 @@ def get_item_by_id(context, item_id):
             first()))
 
 
+def get_item_by_id_cross_account(context, item_id):
+    return (_unpack_item_data(model_query(context, models.Item).
+            filter_by(id=item_id).
+            first(), True))
+
+def get_item_by_os_id_cross_account(context, item_id):
```

Comment:
doses os_id is a unique field ?
do we need to pass kind here to avoid conflict ?